cdc/sink/mq/mq_flush_worker.go

```diff
@@ -201,9 +192,6 @@ func (w *flushWorker) asyncSend(
 // until it encounters an error or is interrupted.
 func (w *flushWorker) run(ctx context.Context) (retErr error) {
 	defer func() {
-		// Note that errCh is sent to exactly once.
-		w.errCh <- retErr
```

Comment:
why remove the retErr?

Reply:
> * remove flush worker errch, because we do not need it. We now send messages to msgchan without blocking anymore.

cc: @nongfushanquan

Reply:
Also, it was supposed to solve the problem of buffer sink getting stuck at that time. It doesn't exist now either.